api/server/server.go

```diff
@@ -800,11 +807,14 @@ func (s *Server) postImagesCreate(eng *engine.Engine, version version.Version, w
 		imageImportConfig.ContainerConfig = newConfig
 
 		if err := s.daemon.Repositories().Import(src, repo, tag, imageImportConfig); err != nil {
-			return err
+			if !output.Flushed() {
```

Comment:
from what I understand is `job.Stdout.Used()` is replaced by `output.Flushed()` it's used to check if we already wrote on the socket, is we did, we cannot return an error (otherwise we will have duplicate write headers and such things)

Here I don't see `job.Stdout.Used()` on the old version so I don't think it's needed. I think `Import` will never write and return an error, either it returns an error or it writes, that why the check isn't needed.


I hope this is understandable.